persist-cli/src/main/java/io/ballerina/persist/cmd/Migrate.java

```diff
@@ -317,6 +314,24 @@ private static void migrateWithTimestamp(File migrationsDir, String migrationNam
                     errStream.println("Error: An error occurred while writing to file: " + e.getMessage());
                     return;
                 }
+
+                String subpath = "";
+                String directoryName = "persist";
+                String pattern = "(?:^|\\/)" + directoryName + "(?:\\/|$)(.*)";
+    
+                Pattern regex = Pattern.compile(pattern);
+                String pathString = newMigrationPath.toString();
+                Matcher matcher = regex.matcher(pathString);
+    
+                if (matcher.find()) {
+                    subpath = directoryName + File.separator + matcher.group(1);
+                } else {
+                    errStream.println("Error: could not find the desired directory.");
+                }
```

Comment:
Can we get the relative path instead of doing the regex matching here.

Reply:
Using https://docs.oracle.com/javase/8/docs/api/java/nio/file/Path.html#relativize-java.nio.file.Path-

Reply:
Compute relative path compare to the current directory